k8s/scripts/provision.sh

```diff
@@ -103,3 +103,20 @@ kubernetesVersion: ${version}
 networking:
   podSubnet: 10.244.0.0/16
 EOF
+
+# Create local-volume directories
+mkdir /var/local/kubevirt-storage
```

Comment:
mkdir -p and combine the two?

Reply:
Been using linux over a decade, never knew that existed... yeah saves a few lines of code.